html-aam/index.html

```diff
@@ -9272,6 +9272,204 @@ <h4 id="att-colspan">`colspan`</h4>
+                <div class="objattrs"><span class="type">Object attributes:</span> `details-roles:popover`</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+                <div class="objattrs"><span class="type">Object attributes:</span> `details-roles:popover`</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <tr>
+              <th>Comments</th>
+              <td>
+                <p>
+                  User Agents MUST expose an <a class="core-mapping" href="#ariaDetails">`aria-details`</a> relationship with the associated element (identified via the specified `commandfor` attribute) <strong>except</strong> under the following conditions:
+                </p>
+                <ul>
+                  <li>The associated popover element is the next immediate accessibility sibling to the invoking element,</li>
+                  <li>The element is a descendant of the `popover` it is associated with.</li>
+                </ul>
+                <p class="note">A button that represents a <a data-cite="html/forms.html#concept-submit-button">submit button</a> with a [=button/form owner=] cannot invoke a command.</p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <h4 id="att-command-dialogs">`command` <span class="att-context">(in the Close and Show Modal states)</span></h4>
+        <table aria-labelledby="att-command-dialogs">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>
+                `command` 
+              </td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td>
+                <a data-cite="html/button.html#attr-input-command">`button`</a>
+                <span class="el-context">(`command` in the 
+                <a data-cite="html/input.html#attr-button-command-close-state">Close state</a>) and
+                <a data-cite="html/input.html#attr-button-command-show-modal-state">Show Modal state</a>)</span>
+              </td>
+            </tr>
+            <tr>
+              <th>[[WAI-ARIA-1.2]]</th>
+              <td>
+                <div class="general">
+                  A `command` attribute in the `close` and `show-modal` states provide no additional accessibility mappings to the `button` element.
```

Comment:
cc @scottaohara is this deliberately omitting the case of a close button pointing to a non-modal dialog (where it *can* be outside of the dialog)?

Reply:
this was deliberate.   if people can provide use cases where this is needed for real ui, then we can always reconsider.  but for now, the general idea would be that close buttons would commonly be within the dialog (modal or not), and if there was a control to close the non-modal dialog, outside of the dialog, it is more likely it'll be a toggle than a singular close button - or the close button might disappear from the DOM - or need to be marked as disabled (since you can't close a closed element), so exposing an expanded state here wouldn't necessarily be useful, either.